tests/publiccloud/nvidia.pm

```diff
@@ -17,8 +17,8 @@ sub validate_nvidia {
     validate_script_output("hwinfo --gfxcard", sub { /nVidia.*Tesla T4/mg });    # depends on terraform setup
         # nvidia-smi is delivered by nvidia-compute-utils-G06. Not available on PublicCloud
         # Check device files and loaded modules instead.
-    validate_script_output("lsmod", sub { m/nvidia/ }, fail_message => "nvidia module not loaded");
-    assert_script_run("ls -la /dev/{nvidia-modeset,nvidia-uvm-tools,nvidiactl,nvidia-uvm,nvidia0}", fail_message => "nvidia device files are missing");
+    validate_script_output("lsmod | grep nvidia", sub { m/nvidia|nvidia_modeset/ }, fail_message => "nvidia module not loaded");
+    #assert_script_run("ls -la /dev/{nvidia-modeset,nvidia-uvm-tools,nvidiactl,nvidia-uvm,nvidia0}", fail_message => "nvidia device files are missing");
```

Comment:
I always against keeping commented code in master , better just delete it or in worst case at least leave a comment why it is commented . Keep in mind that we using git anyway so nothing will be lost even after deletion :)